packages/rrweb/src/record/index.ts

```diff
@@ -41,8 +41,9 @@ import {
 } from './error-handler';
 
 let wrappedEmit!: (e: eventWithoutTime, isCheckout?: boolean) => void;
-
-let takeFullSnapshot!: (isCheckout?: boolean) => void;
+let takeFullSnapshot!: (isCheckout?: boolean) => void = () => {
```

Comment:
you can drop the bang `!` here since you are defining it right away

Reply:
done